### ISSUE TITLE
auto reload is not working after saving the file

### ISSUE THREAD

**Issue by AkashJugnu** [ritwickdey/vscode-live-server#2331]
## I'm submitting a...

```html
[ ] Regression (a behavior that works but auto-reload the page after saving is not working)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Current behavior

- when I start live serve it works but when I save the .HTML file after few changes the automatic reload of page is not working and I have to do every time manually.

## Environment

```html
Browser:

- [✓] Chrome (desktop) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: <!-- version v5.7.3 --> 
- Platform:  <!-- Windows -->
- Visual Studio Code: <!-- `code --version` -->
```

**Comment by hiccup-09** [comment]
> ## I'm submitting a...
> 
> ```
> [ ] Regression (a behavior that works but auto-reload the page after saving is not working)
> [ ] Bug report  
> [ ] Feature request
> [ ] Documentation issue or request
> [ ] Other: 
> ```
> 
> ## Current behavior
> 
>     * when I start live serve it works but when I save the .HTML file after few changes the automatic reload of page is not working and I have to do every time manually.
> 
> 
> ## Expected behavior
> ## Environment
> 
> ```
> Browser:
> 
> - [✓] Chrome (desktop) version XX
> - [ ] Firefox version XX
> - [ ] Safari (desktop) version XX
> - [ ] IE version XX
> - [ ] Edge version XX
> 
> For Tooling issues:
> 
> - Live Server:  
> - Platform:  
> - Visual Studio Code: 
> ```
> 
> ## Others


facing the same issue !!

**Comment by ritwickdey** [comment]
Can you install an older version v5.6.x and just try? let me know if works!

**Comment by ritwickdey** [comment]
...and can you please try in incognito?

**Comment by AkashJugnu** [comment]
I have already tried all the old versions but that also didn't work.
I tried to search on internet and all forum but won't get resolved the
problem. So finally i found you from developer and contact u.

**Comment by ritwickdey** [comment]
Did you try into the incognito window?  and I need a few more information.
1. Was it working previously?
2. Did you change any configuration? (e.g. `liveServer.settings.useWebExt ` should be false or unset)
3. Open Chrome (incognito), Start LiveServer and check if any errors in Chome Console?
4. In the same chrome navigate to the `network` tab, select `WS` (WebSocket) and check if any connection is open or not? (Or share screenshot)

**Comment by mehrzadtajkarimi** [comment]
Hello
I had the same problem
Option GoLive was not
PHP I was working on the file
Lucky you HTML I checked the option was added immediately

**Comment by ritwickdey** [comment]
hi @mehrzadtajkarimi, Just now I've published a new version (v5.7.4) that will fix the PHP issue. GoLive will be visible if any PHP file is opened
